addons/mail/controllers/discuss.py

```diff
@@ -285,3 +285,65 @@ def message_get_suggested_recipients(self, model, res_ids):
         except Exception:
             return {}
         return records._message_get_suggested_recipients()
+
+    # --------------------------------------------------------------------------
+    # RTC API
+    # --------------------------------------------------------------------------
+
+    @http.route('/mail/rtc/session/notify_call_members', type="json", auth="public")
```

Comment:
consider rename, better match between route, method name, and private method name?